pages/residences-parish.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div>
+    <div id="main-content" class="pl-5 pr-5 pt-3 pb-3">
+      <div class="flex content-start items-center">
+        <h1 class="text-sm md:text-xl lg:text-2xl">Number of Accused Witches That Resided In Each Modern Civil Parish</h1>
+        <span class="rounded-full border-r border-l border-gray-400 w-6 h-6 flex items-center justify-center ml-2">
+          <!-- icon by feathericons.com -->
+          <svg aria-hidden="true" class="" data-reactid="266" fill="none" height="24" stroke="#606F7B" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg" @click="showPageInfo()">
+            <line x1="12" y1="16" x2="12" y2="12"></line><line x1="12" y1="8" x2="12" y2="8"></line>
+          </svg>
+        </span>
+      </div>
+    </div>
+    <div>
+      <div class="embed-container-parish">
+        <iframe width="500" height="400" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" title="Parish Boundary Residence Location of Accused Witches" src="//edinuniv.maps.arcgis.com/apps/Embed/index.html?webmap=379dbfae89704e31ba67e1811bc33605&extent=-7.4748,55.2089,0.6386,57.8114&zoom=true&previewImage=false&scale=true&search=true&searchextent=true&legend=true&disable_scroll=false&theme=dark"></iframe>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import Swal from 'sweetalert2';
+
+definePageMeta({
+  layout: 'default'
+});
+
+export default {
+  data: () => ({}),
+  methods: {
+    showPageInfo() {
```

Comment:
This isn't directly related to this PR but I noticed it and thought I'd flag it here for fixing. When we click on the (i), there is no indication that this is clickable. When we hover over it, the cursor should change to be a wee hand.

I think we should update this to use an icon instead. I think we've used feather icons elsewhere so maybe https://feathericons.com/?query=info

Also we need to make sure this icon is accessible so is ignored by a screenreader with a suitable text alternative. Thanks Ruby

Reply:
I've change to used the same infoIcon that the MapFilters component uses and made all of them cursor pointer, i also did this for the arrows that hide/show filter boxes to make that more obvious its clickable. I also made the change for other pages that have an info icon so that it is consistent. Let me know if you prefer the look of the icon you suggested and I can change it to that.